db/migrate/20240421044950_change_email_to_be_citext_in_users.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeEmailToBeCitextInUsers < ActiveRecord::Migration[7.1]
+  def up
+    enable_extension("citext")
+    change_column :users, :email, :citext
+  end
+
+  def down
+    enable_extension("citext")
```

Comment:
Doing same thing in up and down

Reply:
fixed migrations and type changed to citext